core/injector/application_injector.cpp

```diff
@@ -1219,6 +1235,8 @@ namespace {
             [](auto const &injector) {
               return get_parachain_processor_impl(injector);
             }),
+        di::bind<network::PeerView>.to(
+            [](auto const &injector) { return get_peer_view(injector); }),
```

Comment:
Is this needed?  
`injector.create<sptr<network::PeerView>>()` works without `get_peer_view`.
```suggestion
```